tests/fixtures/connection_string/mockups.toml

```diff
@@ -256,6 +282,25 @@ preference = "Leader"
 secure = false
 tls_verify_cert = false
 throw_on_append_failure = true
+keep_alive_interval = 10_000
+keep_alive_timeout = 10_000
 [[mockups.expected.hosts]]
 host = "host"
 port = 2_113
+
+[[mockups]]
+string = "esdb://localhost?keepAliveInterval=20&keepAliveTimeout=10"
```

Comment:
**CONCERN:** Can we add more test scenarios? E.g.
- when values equal to `-1` are provided,
- when values smaller than `-1` are provided,
- plus some permutations with valid and invalid values.

Reply:
Done